### PR TITLE
Update syntax to Swift 1.2

### PR DESCRIPTION
**Pull request by mogstad**

### CODE REVIEW

File: URITemplateTests/URITemplateCases.swift [hunk 1]
```diff
@@ -108,12 +108,12 @@ struct Suite {
   let name:String
   let variables:Dictionary<String, AnyObject>
   let cases:[Case]
-  let level:Int
+  var level:Int
```
Comment (kylef) [inline URITemplateTests/URITemplateCases.swift]: Why would this become `var`, it's never changed?

Comment (mogstad) [inline URITemplateTests/URITemplateCases.swift]: Seems like Swift 1.2 has been a bit more picky, it gets reassigned in `init` on line 121, but as it’s only happening in the there, I can just change it to an else instead.

**Comment by mogstad** [comment]
Let me know if it looks good, and I will rebase it real quick.

**Comment by kylef** [comment]
Sorry for delay in merging, finally got round to downloading Xcode 6.3.

**Pull request merged**